docker-compose.yml

```diff
@@ -14,7 +14,11 @@ services:
     depends_on:
       - "zookeeper"
     environment:
-      KAFKA_CREATE_TOPICS: "user_names:4:1,user_follow_infos:4:1,user_scrape_errors:1:1,reached_limit:4:1,renewed_elastic_ip:4:1,renewing_elastic_ip_errors:4:1,user_post:4:1,user_instagram-scrape_errors:4:1,insta_comments_info:4:1,post_comment_insta_scraper_errors:4:1"
+      KAFKA_CREATE_TOPICS: >
+        user_names:4:1,user_follow_infos:4:1,user_scrape_errors:1:1,
+        twitter-user_names:4:1,twitter-users_scraped:4:1,twitter-posts_scraped:4:1,
+        reached_limit:4:1,renewed_elastic_ip:4:1,renewing_elastic_ip_errors:4:1,
+        user_post:4:1,user_instagram-scrape_errors:4:1,insta_comments_info:4:1,post_comment_insta_scraper_errors:4:1,
```

Comment:
`post_comment_insta_scraper_errors:4:1,`
this is the reason for https://github.com/codeuniversity/smag-mvp/issues/30